python/sglang/srt/entrypoints/http_server.py

```diff
@@ -814,9 +815,38 @@ def _wait_and_warmup(
                 timeout=600,
             )
             assert res.status_code == 200, f"{res}"
+        elif server_args.disaggregation_mode == "prefill":
+            logger.info(f"Start of prefill warmup ...")
+            json_data = {
+                "sampling_params": {
+                    "temperature": 0.0,
+                    "max_new_tokens": 8,
+                    "ignore_eos": True,
+                },
+                "bootstrap_host": [FakeBootstrapHost] * server_args.dp_size,
+                # This is a hack to ensure fake transfer is enabled during prefill warmup
+                # ensure each dp rank has a unique bootstrap_room during prefill warmup
+                "bootstrap_room": [
```

Comment:
why does it matter if we are using fake sender?

Reply:
I want to use it to trigger the DeepGEMM precache operation under each DP (Data Parallel) rank,
because the precache mechanism mentioned earlier (https://github.com/sgl-project/sglang/pull/5580) still requires CUDA Graph to be enabled in order to trigger DeepGEMM warmup.
However, since prefill does not enable CUDA Graph, I need to rely on this warmup request to perform the precache.